support-module/src/main/java/com/saudigitus/support_module/ui/MenuScreen.kt

```diff
@@ -12,40 +14,61 @@ import androidx.compose.runtime.Composable
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.rememberNavController
 import com.saudigitus.support_module.R
 import com.saudigitus.support_module.ui.components.CustomCard
-
+import android.content.Context
+import android.content.Intent
+import com.saudigitus.support_module.MainActivity
 
 @Composable
-fun MenuScreen() {
+fun MenuScreen(context: Context) {
     Column(
         modifier = Modifier
             .fillMaxWidth()
             .height(170.dp)
-            .background(color = Color(0xFFFFFFFF))
-            .padding(5.dp),
+            .background(color = Color(0xFFFFFFFF)),
         verticalArrangement = Arrangement.Top,
         horizontalAlignment = Alignment.Start
     ) {
+        Spacer(modifier = Modifier.height(8.dp))
         Row(
             modifier = Modifier
                 .fillMaxSize()
-                .padding(16.dp),
+                .padding(10.dp),
             horizontalArrangement = Arrangement.SpaceEvenly,
             verticalAlignment = Alignment.CenterVertically
         ) {
             // First Card
-            CustomCard(imageResId = R.drawable.manual_icon, title = R.string.manuals.toString())
+            CustomCard(
+                imageResId = R.drawable.manual_icon,
+                title = stringResource(id = R.string.manuals),
+                onClick = {
+                    val intent = Intent(context, MainActivity::class.java)
+                    intent.putExtra("screen", Screen.Manuals.route)
```

Comment:
create constant file and sent key constants
```suggestion
                    intent.putExtra("screen", Screen.Manuals.route)
                    intent.putExtra("screen", Screen.Support.route)
```

Reply:
Done